base/compiler/tfuncs.jl

```diff
@@ -2680,80 +2681,145 @@ _iszero(@nospecialize x) = x === Intrinsics.xor_int(x, x)
 _isneg1(@nospecialize x) = _iszero(Intrinsics.not_int(x))
 _istypemin(@nospecialize x) = !_iszero(x) && Intrinsics.neg_int(x) === x
 
-function intrinsic_nothrow(f::IntrinsicFunction, argtypes::Vector{Any})
+
+
+function builtin_exct(𝕃::AbstractLattice, @nospecialize(f::Builtin), argtypes::Vector{Any}, @nospecialize(rt))
+    if isa(f, IntrinsicFunction)
+        return intrinsic_exct(𝕃, f, argtypes)
```

Comment:
Now there is a duplicated call to `intrinsic_exct` in `builtin_nothrow`, so we would probably want to unify the nothrow analysis and the exct modeling in the future.

Reply:
Yes, it's a bit unfortunate, but I think we'll probably just remove the nothrow effect in the future in favor of exct everywhere.